Index: AR/ar.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\n<html lang=\"en\" >\n<head>\n    <script src=\"aframe.min.js\"></script>\n    <script src=\"aframe-ar.js\"></script>\n</head>\n<body>\n<a-scene embedded arjs>\n\n    <a-marker preset=\"hiro\" id=\"marker1\">\n        <a-box id=\"MyBox\" position='0 0.5 0' material='opacity: 0.5;' color=\"red\" ></a-box>\n    </a-marker>\n\n    <a-marker type=\"pattern\" preset=\"pattern\" url=\"https://raw.githubusercontent.com/NicolaMenale/AR-for-Education/main/textures/pattern-marker2.patt\" id=\"marker2\">\n        <a-sphere id=\"MySphere\" position='0 0.5 0' material='opacity: 0.5;' color=\"red\"></a-sphere>\n    </a-marker>\n\n    <a-entity camera></a-entity>\n</a-scene>\n\n<script src=\"AR_Principale/assets/js/script_ar.js\"></script>\n</body>\n</html>\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AR/ar.html b/AR/ar.html
--- a/AR/ar.html	(revision 8c18becf5e5808c10651e709ed275663b2ade412)
+++ b/AR/ar.html	(date 1701960428406)
@@ -18,7 +18,7 @@
     <a-entity camera></a-entity>
 </a-scene>
 
-<script src="AR_Principale/assets/js/script_ar.js"></script>
+<script src="../AR_Principale/assets/js/script_ar.js"></script>
 </body>
 </html>
 
